keepsorted/options.go

```diff
@@ -242,8 +242,8 @@ func (opts blockOptions) maybeParseNumeric(s string) numericTokens {
 				// See the comment on numericTokens for more details.
 				t.s = append(t.s, "")
 			}
-			i, err := strconv.ParseInt(sm[1], 10, 64)
```

Comment:
Oh, yeah, that was a downside of starting with Python and converting to Go. It presumably used to handle these numbers fine and regressed D: